etc/build-binary.bash

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+set -x
+
+KIND=${1:-production}
+DIR="$(cd -P "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+BASE=${DIR}/..
+
+if [ ${var} ]
```

Comment:
I think this is wrong.  `var` is not defined until line 29, so unless `var` is passed from the environment, this test always fails.